nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -88,6 +88,8 @@
 
 - The `go-ethereum` package has been updated to v1.12.0. This drops support for proof-of-work. Its GraphQL API now encodes all numeric values as hex strings and the GraphQL UI is updated to version 2.0. The default database has changed from `leveldb` to `pebble` but `leveldb` can be forced with the --db.engine=leveldb flag. The `checkpoint-admin` command was [removed along with trusted checkpoints](https://github.com/ethereum/go-ethereum/pull/27147).
 
+- `libpulsar` Python wrapper support has been removed and is now a standalone Python package `pythonPackages.pulsar`. See new standalone [repository](https://github.com/apache/pulsar-client-python) for more details.
```

Comment:
It should be enough to add an alias for this or are there any other changes that people must know?

Reply:
It should be enough.

Reply:
So we can drop the entire changelog entry and add the alias or why not?

Reply:
please see https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix

Reply:
Oh I see but how can I restrict it in usage of a "Python" derivation?

Reply:
I am not sure what you want to restrict.